docs/backend/setup/database-migrations.md

```diff
@@ -158,6 +159,15 @@ Migrations are run by the `milmove migrate` command. This allows us to leverage
 
 The reason to use a `make` target is because it will correctly set the migration flag variables and target the correct database with environment variables.
 
+## Delete Migrations
```

Comment:
Wait so is this saying to delete the migration file itself, or only remove the line in the manifest that points to it?

Mostly because if it's the latter, then to me it's not so much about "deleting" a migration, as making edits and re-running a migration, in which case running the `make db_dev_reset db_dev migrate` command would handle it.

Reply:
Delete the migration file itself. Although if there is a better way to delete a migration, I'd rather offer that option...

Reply:
I think I failed to explain this well when we paired. You only need to remove the file from the manifest, you don't need to delete the migration file. If you do delete the file the build will complain of a missing migration since it would still be in the manifest.

I would also add a note that this is only if you want to undo a migration you've added locally. For the deployed environments the process is to create a new migration that un-does the change.

Let me know if this helps if not I'm happy to connect on Monday :)

Reply:
That's what I meant. The migration file that is in the manifest, not the entire folder. Does that clear things up or is there another migration file that I didn't understand?

Reply:
I think maybe in that case this should be split into two sections. Like one for running a changed migration (that hasn't made it to a deployed env). Which I think would be two steps:

1. Make changes to migration
2. Run

    ```shell
    make db_dev_reset db_dev migrate
    ```

And a section on deleting a migration fully. This one could have two things, one about it being local only still, so 3 steps:

1. Remove the line in the manifest pointing at the migration file.
2. Delete the migration file
3. Run 

    ```shell
    make db_dev_reset db_dev migrate
    ```

And a part mentioning that if doing it for a migration that has already been run against the deployed environments, that we need to follow our zero-downtime migrations practice (which is on this page).

Reply:
Do we ever delete migrations once they have been deployed? Not sure that is something we need to document, at least until we actually need to do so.